src/Tintin/Html/Templating.hs

```diff
@@ -169,9 +173,10 @@ tintinHeader info@Project.Info {..} Project.Page {..} =
     meta_ [ itemprop_ "og:title"
           , content_ (name <> " - " <> title)
           ]
-    meta_ [ itemprop_ "og:url"
-          , content_ githubLink
-          ]
+    when (isJust githubLink) $
+      meta_ [ itemprop_ "og:url"
+            , content_ (fromJust $ githubLink)
+            ]
```

Comment:
Same here @J2RGEZ

Reply:
Would it make sense to provide a more generic author and url option so tintin renders whatever that the user decides? Github authors can still put the GH URL there and other users can link their own websites or whatever they want, and they'll also be able to write their names in any way they want... what do you think?

Reply:
The problem here is the Open Graph "standard". In their website the basic operations are these ones:
![image](https://user-images.githubusercontent.com/7570085/40657236-00e6e7ea-633f-11e8-9da6-f3eb3f8a9791.png)

So I think the Open Graph card will be affected by this.

Reply:
Well, my proposal would be that instead of generating the URL out of the github user name as we're doing now, we can configure tintin with a full URL, so it's just a matter of pasting the right one. Another option is having two config fields: the current one and an extra "fallback" url that would be used in case that the GH name is not providen. What do you think?